### ISSUE TITLE
Deprecated example in the user guide: Asynchronous monitoring

### ISSUE THREAD

**Issue by denilsonsa** [pyudev/pyudev#155]
Excerpt from <https://pyudev.readthedocs.org/en/latest/guide.html#asynchronous-monitoring>:

```python
>>> monitor = pyudev.Monitor.from_netlink(context)
>>> monitor.filter_by('block')
>>> def log_event(action, device):
...    if 'ID_FS_TYPE' in device:
...        with open('filesystems.log', 'a+') as stream:
...            print('{0} - {1}'.format(action, device.get('ID_FS_LABEL')), file=stream)
...
>>> observer = pyudev.MonitorObserver(monitor, log_event)
>>> observer.start()
```

But, according to <https://pyudev.readthedocs.org/en/latest/api/pyudev.html#pyudev.MonitorObserver.__init__>:

> `__init__(monitor, event_handler=None, callback=None, *args, **kwargs)`
>
> […] callback is the callable to invoke on events, with the signature **`callback(device)`** […]
>
> Deprecated since version 0.16: The `event_handler` argument will be removed in 1.0. Use the callback argument instead.

This means that the example code in the user guide is using the deprecated `event_handler` argument, which uses a different callback function signature.

**Comment by mulkieran** [comment]
Thanks. Do you feel like preparing a PR for this? Ideally, the docs should show the old way, since it is still OK until 1.0, point out that it is deprecated, and then show the new and better way.

**Comment by mulkieran** [comment]
Fixed in 0.21.
